pkg/justification/processor_test.go

```diff
@@ -109,6 +109,13 @@ func TestCreateToken(t *testing.T) {
 			},
 			wantErr: "failed to validate request",
 		},
+		{
+			name: "ttl_too_long",
+			request: &jvspb.CreateJustificationRequest{
+				Ttl: durationpb.New(25 * time.Hour),
+			},
+			wantErr: "failed to validate request",
```

Comment:
Should we use the new error message?

Reply:
+1 can we add more specific wantErr substring in UT?